build_scripts/build_usd.py

```diff
@@ -1114,6 +1271,11 @@ def InstallTIFF(context, force, buildArgs):
                    [("add_subdirectory(tools)", "# add_subdirectory(tools)"),
                     ("add_subdirectory(test)", "# add_subdirectory(test)")])
 
+        if context.targetIos:
```

Comment:
tbh, my opinion is that we should skip contrib on all platforms as people don't look to the USD project to build libtiff for them unless they are simply satisfying requirements of the oiio build... Not requesting an update though, as it's an existing behavior that contrib gets roped in.

Reply:
In our initial submission we could skip supporting OpenImageIO, although it would be nice - are you aware if any of OpenColorIO's dependencies are optional?

Reply:
errrmmm, 99% of them are? There are a lot of vendor package inclusions, documentation generators, Python dependencies, and OpenShadingLanguage. But I don't think we have to take particular action on that on behalf of iOS. On Mac, OCIO uses pybind11, not boost python thank goodness, so we are clean on the OCIO + boost front, as far as I can tell.